microsoft-edge/webview2/concepts/enterprise.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Understand how to manage WebView2 applications
+title: Managing WebView2 applications
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/15/2020
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html, enterprise, group policy, manageability 
+---
+
+# Managing WebView2 applications  
+
+[WebView2][WebView2Landing] is a component that developers use to build their applications, and the developers may deploy a [self-updating Evergreen WebView2 Runtime][Webview2ConceptsDistributionUnderstandRuntimeInstallerPreview] on user device to power their applications. This document discusses how IT admins may manage WebView2 applications and Runtime. Feedback from both IT admins and developers are welcomed on [WebView2 Feedback repo][GithubMicrosoftedgeWebviewfeddback].  
```

Comment:
```suggestion
[WebView2][WebView2Landing] applications may depend on the [self-updating Evergreen WebView2 Runtime][Webview2ConceptsDistributionUnderstandRuntimeInstallerPreview], which means the runtime can be installed on user devices. This document discusses how IT admins may manage WebView2 applications and the runtime. Feedback from both IT admins and developers is welcome on the [WebView2 Feedback repo][GithubMicrosoftedgeWebviewfeddback]. 
```

Reply:
Would probably leave the 1st couple of sentences as-is. I'd like to start the the doc describing what WV2/WVRT is and how they get onto the devices maintained by admins. 

Will fix the last sentence.